package/kedro_viz/api/graphql.py

```diff
@@ -239,16 +240,18 @@ def get_run_tracking_data(
         all_runs = {}
         for run_id in run_ids:
             run_id = ID(run_id)
-            file_path = dataset._get_versioned_path(str(run_id))
-            if Path(file_path).is_file():
+            # Set the load_version to run_id
+            dataset._version = DataSetVersion(run_id, None)
```

Comment:
@rashidakanchwala do you mind helping me understand why it fixed the issue?

Reply:
So removing the if Path(file_path).is_file() fixed it because it was returning false. 

The other stuff I just changed because Antony suggested we should use load method instead of directly accessing private function. 

I did some debugging around why Path(file_path).is_file is returning false when we use Win CMD. Too long to put it here, so I will share with u on discord. 

P.S - miss you on slack